irohad/model/converters/impl/pb_block_factory.cpp

```diff
@@ -15,8 +15,10 @@
  * limitations under the License.
  */
 
-#include <boost/assert.hpp>
 #include "model/converters/pb_block_factory.hpp"
+
+#include <boost/assert.hpp>
+
```

Comment:
Minor
The blank line is not needed. Only stdlib headers should be separated like that.